index.js

```diff
@@ -477,7 +370,7 @@ function studentDebtCalculator( financials ) {
   // Institutional Loan debt at graduation
   data.institutionalLoanTotal = calcDebt( data.institutionalLoan,
     data.institutionalLoanRate, data.programLength, data.deferPeriod );
-      
+
   // Home Equity Loans at graduation
   data.homeEquityTotal = (data.homeEquity * .079 / 12 * ((data.programLength * (data.programLength + 1) / 2 * 12)));
```

Comment:
We might not need an explicit home equity loan at all. It might be better to bundle any private loans into an object with loan amount and rate (and deferral period?) And iterate through that object.

I hadn't really touched this part because I thought it might be irrelevant...